src/drivers/Kconfig

```diff
@@ -20,6 +20,20 @@ config DUMMY_DMA
 
 	  If unsure, select "n".
 
+config IMX_IRQ
+	bool "i.MX IRQ"
+	default n
+	depends on IMX
+	help
+	  Select this to enable support for i.MX interrupt with irq.
+
+config IMX_NOIRQ
+	bool "i.MX NO IRQ"
+	default n
+	depends on IMX
+	help
+	  Select this to enable support for i.MX interrupt without irq.
+
 config IPC_POLLING
```

Comment:
I think for now IMX platforms fall into 2 categories
- with irqsteer -> people should select CONFIG_IMX_INTERRUPT_IRQSTEER
- withouht irqsteer -> I think here we should add an option named CONFIG_IMX_GENERIC_INTERRUPT

and that's all. IMX_IRQ and IMX_NOIRQ are not really two options. Is just one option and the other one is negation of first.

Reply:
Great, so it can be made a single boolean then.